Sources/HummingbirdWSCore/WebSocket.swift

```diff
@@ -276,8 +335,26 @@ extension HBWebSocket {
     }
 }
 
-#if compiler(>=5.6)
 // HBWebSocket can be set to Sendable because ping data which is mutable is
 // managed internally and is only ever changed on the event loop
 extension HBWebSocket: @unchecked Sendable {}
-#endif // compiler(>=5.6)
+
+/// Extend WebSocketFrame to conform to CustomDebugStringConvertible
+extension WebSocketFrame: CustomDebugStringConvertible {
```

Comment:
While I like it, doesn't this fall under "Don't extend types you don't own with protocols you don't own"?

Reply:
Fair point. I think I added these when debugging the code. Maybe I should open a PR to swift-nio with a version of this